src/platforms/react-native/manual-setup/codepush.mdx

```diff
@@ -81,4 +81,17 @@ If you don't want to use the automatically computed release format, you will nee
 sentry-cli react-native appcenter {APP} ios ./build/CodePush --deployment {DEPLOYMENT} --dist {DIST} --release-name {RELEASE_NAME}
 ```
 
+<Alert level="info">
+
+Make sure to export the `APPCENTER_ACCESS_TOKEN` in your environment in order to run the `appcenter` commands above. Ohterwise you might get an empty error:
```

Comment:
```suggestion
Make sure to export the `APPCENTER_ACCESS_TOKEN` in your environment in order to run the `appcenter` commands above. Otherwise, you might get an empty error:
```